plugins/synaptics-rmi/fu-synaptics-rmi-firmware.c

```diff
@@ -40,8 +40,18 @@ G_DEFINE_TYPE(FuSynapticsRmiFirmware, fu_synaptics_rmi_firmware, FU_TYPE_FIRMWAR
 #define RMI_IMG_V10_CNTR_ADDR_OFFSET 0x0c
 #define RMI_IMG_MAX_CONTAINERS	     1024
 
+/* nocheck:name */
 static gboolean
-fu_synaptics_rmi_firmware_add_image(FuSynapticsRmiFirmware *self,
+fu_synaptics_rmi_firmware_parse_sbl_container_v10(FuFirmware *firmware,
```

Comment:
I think this should be `FuSynapticsRmiFirmware *self`

Reply:
It is fixed in latest commit.